babel.config.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * This is a project wide babel configuration.
+ * https://babeljs.io/docs/en/config-files#project-wide-configuration
+ *
+ * We use this file to apply babel configuration to packages in `node_modules`
+ * for testing with jest.
+ */
+module.exports = {
+  env: {
+    test: {
+      presets: [
+        ["@babel/env", { targets: "last 2 versions, ie 11", modules: false }],
+        "@babel/react",
+      ],
+    },
+  },
+};
```

Comment:
Is it a hard ask for you to move this file to a non-root location? Possibly inside `src/core/build`?

Reply:
Yes, there is literally no options to configure this, besides this is only needed for testing and I think shoudn't be in `src/core/build`